### ISSUE TITLE
Baudrate defaults to 115000 instead of 115200

### ISSUE THREAD

**Issue by aaugustin** [civic/elitech-datareader#19]
I'm not sur if that makes a difference in practice, but the source code uses 115000 in three places while the theoretical value is 115200.

I noticed this while investigating reliability issues with e RC-5 (ie. I'm often not managing to read from the device — I haven't isolated the cause yet).

**Comment by aaugustin** [comment]
To conclude my investigations on baudrate, I tried a bunch of values and nothing other than 115200 works with a RC-5 on macOS.

115000 seems to be interpreted as 115200, so it doesn't make a difference in practice, but it would be cleaner to use the correct value.

For the record, I solved by reliability problems by _not_ calling `device.init()`. I ran dtrace on the Rc Logger app and it appears to jump straight to the `device.get_devinfo()` request. The `device.init()` request didn't receive a response (its result was ignored anyway) and seemed to confuse the device.
